backend/composer/management/commands/ingest_anatomical_entities.py

```diff
@@ -1,64 +1,88 @@
 import csv
-
-from django.core.management.base import BaseCommand
-
-from composer.models import AnatomicalEntity
+import time
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from django.db.utils import IntegrityError
+from composer.models import AnatomicalEntity, Synonym
 
 URI = "o"
 NAME = "o_label"
 SYNONYM = "o_synonym"
+BULK_LIMIT = 100
 
 
 class Command(BaseCommand):
-    help = "Ingests Anatomical Entities CSV file(s)"
+    help = "Ingests Anatomical Entities CSV file(s), with robust error handling."
 
     def add_arguments(self, parser):
         parser.add_argument("csv_files", nargs="+", type=str)
+        parser.add_argument("--update_names", action='store_true',
```

Comment:
question: @afonsobspinto what is the use case for this option? I don't see any benefit of setting this to False

Reply:
👍